etc/lfrc.example

```diff
@@ -45,9 +48,12 @@ cmd open &{{
     esac
 }}
 
-# define a custom 'rename' command without prompt for overwrite
-# cmd rename %[ -e $1 ] && printf "file exists" || mv $f $1
-# map r push :rename<space>
+cmd mkdir %{{
+    IFS=" "
+    mkdir -p -- "$*" &&
```

Comment:
I feel like it would be better to leave the default IFS and use `"$1"` instead of `"$*"`. If you want spaces in the filename you should quote it, same as you would at the shell or with other lf commands like cd. Also with `"$*"` you still have to quote non-space special characters and multiple spaces get collapsed to a single one.

Reply:
What is the default IFS? `lfrc.example` has `set ifs "\n"` at the top. Is that the one you mean? I mainly want to double-check that just deleting the IFS line here is OK.

I'm not a shell expert, so the relationship between `"$*"` and IFS is confusing to me; I just copied this from the wiki.

Unless somebody disagrees, I'll implement your suggestion and change the mapping below to `map C push :mkdir<space>""<left>`.

Reply:
`"$*"` will join all the arguments with the first character of IFS. Otherwise IFS doesn't matter here.

Reply:
OK, I edited it as you suggested. Thanks!